SSO MFA for Admin MFA docs (#67149)
gravitational/teleport: docs/pages/zero-trust-access/authentication/mfa-for-admin-actions.mdx

(!docs/pages/includes/edition-prereqs-tabs.mdx!)

- (!docs/pages/includes/tctl.mdx!)

> - A second factor that can satisfy admin actions configured on the cluster; either of the following:

Reviewer: ```suggestion
- The following MFA methods can satisfy admin actions configured on the cluster:
```

  - [WebAuthn](../management/security/idp-compromise.mdx) with a hardware
    authenticator (such as YubiKey or SoloKey) and a browser with
    [WebAuthn support](https://developers.yubico.com/WebAuthn/WebAuthn_Browser_Support/)
  - [SSO MFA](../sso/sso-for-mfa.mdx), which delegates the MFA check to your
    OIDC or SAML identity provider.

## Require MFA for administrative actions

MFA for administrative actions is automatically enforced on clusters whose

> configured second factors all satisfy admin actions — that is, OTP is not

Reviewer: ```suggestion
configured multi-factors all satisfy admin actions — that is, OTP is not
```

> allowed. WebAuthn and SSO MFA both qualify, individually or together.

Reviewer 2: Could we state in a clearer manner that WebAuthn and SSO MFA are eligible for admin actions but OTP is not.
